jumpy/requirements.txt

```diff
@@ -1,4 +1,9 @@
+attrs==18.1.0
 Cython==0.28.2
+more-itertools==4.1.0
```

Comment:
Excuse my ignorance, but I don't see (from your PR) why we need most of these new requirements. @akshaysharma096

Reply:
@maxpumperla ,  `more-itertools` is in the dependency of pytest testing suite, we can verify from https://github.com/pytest-dev/pytest/blob/master/setup.py.

Thanks!

Reply:
exactly, but you get them with pytest. no need for attr etc.